src/test/java/org/example/GetAllProductList.java

```diff
@@ -16,31 +16,27 @@
 import static io.restassured.RestAssured.*;
 
 public class GetAllProductList {
-
     private static String url;
     private String productId;
     private String productName;
     private String productPrice;
     private String productBrand;
 
     public GetAllProductList() {
-
         try {
             url = ConfigReader.getUrl();
         } catch (IOException e) {
             throw new RuntimeException(e);
```

Comment:
Add loggers. Instead of throwing exception.

Reply:
Done